lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -20,6 +20,7 @@ class SemanticHighlighting < BaseRequest
       TOKEN_TYPES = [
         :variable,
         :method,
+        :namespace,
```

Comment:
Since we can't differentiate between a module and class reference, should we use namespace or class?

Reply:
I thought calling both module and classes a class token may be confusing, and so settled with namespace. Any opinions?